sqlserver/tests/test_statements.py

```diff
@@ -181,26 +194,35 @@ def test_statement_metrics_and_plans_windows(
         query,
         param_groups,
         match_pattern,
+        caplog,
     )
 
 
 def _run_test_statement_metrics_and_plans(
-    aggregator, dd_run_check, dbm_instance, bob_conn, database, plan_user, query, param_groups, match_pattern
+    aggregator, dd_run_check, dbm_instance, bob_conn, database, plan_user, query, param_groups, match_pattern, caplog
 ):
+    caplog.set_level(logging.INFO)
     check = SQLServer(CHECK_NAME, {}, [dbm_instance])
 
-    with bob_conn.cursor() as cursor:
-        cursor.execute("USE {}".format(database))
-
     def _run_test_queries():
         with bob_conn.cursor() as cursor:
+            cursor.execute("USE {}".format(database))
             for params in param_groups:
+                logging.info("running query %s: %s", query, params)
```

Comment:
Is this logging just for tests? If so, it might not be necessary to keep.

Reply:
Yes it's only for the tests. It's useful for when the tests have issues. If the tests are passing then the message will never be seen.